src/FigmaSharp.Maui.Graphics.Sample/MainPage.xaml.cs

```diff
@@ -1,10 +1,55 @@
-﻿namespace FigmaSharp.Maui.Graphics.Sample
+﻿using FigmaSharp.Maui.Graphics.Sample.ViewModels;
+
+namespace FigmaSharp.Maui.Graphics.Sample
 {
     public partial class MainPage : ContentPage
     {
+        private MainViewModel VM;
+
         public MainPage()
         {
             InitializeComponent();
         }
+
+        protected override void OnBindingContextChanged()
+        {
+            base.OnBindingContextChanged();
+
+            if (BindingContext is MainViewModel vm)
+            {
+                VM = vm;
+                vm.DrawableSet += VmOnDrawableSet;
```

Comment:
Could you unsubscribe the event? (For example, can use the `Page.OnAppearing` and `Page.OnDisappearing`.

Reply:
Hello, changed to OnAppearing and OnDisappearing as requested